lib/webidl2.js

```diff
@@ -72,11 +98,11 @@ function parseByTokens(tokeniser, options) {
       callback() ||
       interface_() ||
       partial() ||
-      Dictionary.parse(tokeniser) ||
+      Dictionary.parse(tokeniser, { ...options?.extensions?.dictionary }) ||
```

Comment:
```suggestion
      Dictionary.parse(tokeniser, options?.extensions?.dictionary) ||
```

Probably no need to spread at all. Same for namespace below.